src/commands/upload.js

```diff
@@ -2,12 +2,11 @@ const constants = require('../constants');
 const utils = require('../utils');
 
 const upload = context => {
-  const zipPath = constants.BUILD_PATH;
   context.line('Preparing to upload a new version.\n');
-  return utils.upload(zipPath, '.').then(() => {
+  return utils.upload().then(() => {
```

Comment:
I think this `upload()` call will result in an error. The third param (`appDir`) is used directly in `path.resolve()`, which only accepts strings and throws on `undefined`. Could use a default param (we're using babel, might as well use some fancy features) or give it a `|| ''` [here](https://github.com/zapier/zapier-platform-cli/pull/321/files#diff-93af3604864045d869b58d65af952dedR264).

Reply:
Oh, nice catch! I only tested with `zapier push` and `zapier build`, not `zapier upload`! 🤦‍♂️